docs/configuration.md

```diff
@@ -34,18 +34,19 @@ a gateway spread requests equally among them (using weight 1 for every node):
 ```shell
 $ neofs-s3-gw -p 192.168.130.72:8080 -p 192.168.130.71:8080
 ```
+
 If you want some specific load distribution proportions, use weights and priorities, they
 can only be specified via environment variables or a configuration file.
 
 ### Wallet
 
-Wallet (`--wallet`) is a mandatory parameter. It is a path to a wallet file. You can provide a passphrase to decrypt 
+Wallet (`--wallet`) is a mandatory parameter. It is a path to a wallet file. You can provide a passphrase to decrypt
 a wallet via env variable or conf file, or you will be asked to enter a password interactively.
 You can also specify an account address to use from a wallet using the `--address` parameter.
 
 ### Listening on address and TLS
 
-Gateway listens on `0.0.0.0:8080` by default, and you can change that with the `--listen_address` option.
+You can make the gateway listen on specific address using the `--listen_address` option.
```

Comment:
I agree that this wording sounds better, but why you decided to remove a mention to a default value?